src/components/Form.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
 
 export default function Form() {
+  const dispatch = useDispatch();
   return (
     <div>
       <section className="mx-44 mt-11">
         <h2>ADD NEW BOOK</h2>
-        <form action="">
+        <form action="" onSubmit={() => dispatch(addBook({ title: '', author: '', id: '' }))}>
           <input type="text" className="border border-grey-900 h-10" />
```

Comment:
- [ ] Please kindly take note that I was unable to add a book to the list as expected, kindly fix this in order to proceed. 👍